### PR TITLE
Schedule next Cortex releases

### PR DESCRIPTION
**Pull request by pracucci**
**What this PR does**:
In this PR I'm proposing next Cortex releases schedule. 1.10 has been scheduled 6 weeks after from 1.9 release candidate, which was delayed by few weeks compared to the original schedule.

**Which issue(s) this PR fixes**:
N/A

**Checklist**
- [ ] Tests updated
- [ ] Documentation added
- [ ] `CHANGELOG.md` updated - the order of entries should be `[CHANGE]`, `[FEATURE]`, `[ENHANCEMENT]`, `[BUGFIX]`

### CODE REVIEW

**Review by gouthamve — APPROVED** [review]
LGTM

File: RELEASE.md [hunk 1]
```diff
@@ -25,6 +25,9 @@ Our goal is to provide a new minor release every 6 weeks. This is a new process
 | v1.7.0         | 2021-01-18                                 | Ken Haines (@khaines)                       |
 | v1.8.0         | 2021-03-01                                 | Peter Štibraný (@pstibrany)                 |
 | v1.9.0         | 2021-04-12                                 | Goutham Veeramachaneni (@gouthamve)         |
+| v1.10.0        | 2021-06-06                                 | Marco Pracucci (@pracucci)                  |
```
Comment (pracucci) [inline RELEASE.md:28]: I'm not doing it since some time. I volunteer for that.

File: RELEASE.md [hunk 2]
```diff
@@ -25,6 +25,9 @@ Our goal is to provide a new minor release every 6 weeks. This is a new process
...
+| v1.11.0        | 2021-07-18                                 |                                             |
```
Comment (bboreham) [inline RELEASE.md:29]: I can do this one

**Review by bboreham — APPROVED** [review]

File: RELEASE.md (hunk 2 above)
Comment (pracucci) [reply RELEASE.md]: Thanks! Assigned to you.

**Pull request merged**